### ISSUE TITLE
Unable to deserialize structure members due to an AttributeError being raised

### ISSUE THREAD

**Issue by arizvisa** [arizvisa/ida-minsc#109]
When deserializing a `structure_t` that was serializing via pickle or anything else, an exception gets raised by the `member_t.__setstate__` method. This looks like the following at the REPL.

```
Python>b=pickle.loads(a)
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib64/python2.7/pickle.py", line 1388, in loads
    return Unpickler(file).load()
  File "/usr/lib64/python2.7/pickle.py", line 864, in load
    dispatch[key](self)
  File "/usr/lib64/python2.7/pickle.py", line 1223, in load_build
    setstate(state)
  File "/home/user/work/ida-minsc/base/structure.py", line 1491, in __setstate__
    logging.info(u"{:s}({:#x}): Field at {:+#x} of structure ({:s}) contains the same name \"{:s}\".".format('.'.join([__name__, cls.__name__]), self.id, ofs, parentname, utils.string.escape(name, '"')))
  File "/home/user/work/ida-minsc/base/structure.py", line 1521, in id
    return self.ptr.id
  File "/home/user/work/ida-minsc/base/structure.py", line 1517, in ptr
    return self.__parent.ptr.get_member(self.__index)
AttributeError: _member_t__index
```

This is actually happening because of a recursion issue when trying to display an error message.

**Comment by arizvisa** [comment]
This is fixed by PR #110.

**Comment by arizvisa** [comment]
Closing this issue as PR #110 was merged.
